addusers.py

```diff
@@ -171,6 +171,50 @@ def grant_role(self, auth_url, project_id, user_id, role_id):
         return self.keystone.session.put(url)
 
 
+def manual_testing(header):
+    """Testing changes to the code that doesn't involve the form
+    """
+    print "...TESTING..."
+    tests = []
+    tests.append(header)
+    num_tests = int(raw_input("Number of tests: "))
+    for idx in range(num_tests):
+        index_length = len(header)
+        index_flag = raw_input("New Project?    [y/n] ")
+        if index_flag == 'y':
+            index_length -= 4
```

Comment:
I agree I do always want the end to be in the same column.

The reason why I made this edit was because the list automatically created from Google Sheets is either length of 17 or 21. Therefore, I thought I should keep things consistent here to have manual inputs in the same structure.